packages/headless/src/controllers/core/facets/facet/headless-core-facet-options.ts

```diff
@@ -62,7 +62,7 @@ export interface FacetOptions {
   numberOfValues?: number;
 
   /**
-   * The criterion to use for sorting returned facet values.
+   * The criterion to use for sorting returned facet values. The default value of `automatic` adjusts the sort based on whether the facet is expanded or not, and whether the full list of values is being displayed.
```

Comment:
The intent of explaining inline is good, but I feel the extra text is still vague. If I was trying to understand the behaviour of `automatic`, I still would not understand after reading the additional text.

I think an clear explanation has to include when the values will be ordered alphanumerically vs. by score.

While we could inline this info, I think linking to the [search api docs](https://docs.coveo.com/en/1461/build-a-search-ui/query-parameters#RestFacetRequest-sortCriteria) could be better to keep a single source of truth.